config/params/config.go

```diff
@@ -438,9 +439,24 @@ func (b *BeaconChainConfig) MaxBlobsPerBlock(slot primitives.Slot) int {
 	if epoch >= b.ElectraForkEpoch {
 		return b.DeprecatedMaxBlobsPerBlockElectra
 	}
+
 	return b.DeprecatedMaxBlobsPerBlock
 }
 
+// AbsoluteMaxBlobsPerBlock returns the absolute maximum number of blobs per block.
```

Comment:
```suggestion
// AbsoluteMaxBlobsPerBlock returns the absolute maximum number of blobs per block from all forks, including forks that have not occurred yet.
```

I think it's important to explain this is not fork aware and could be an issue for some use cases then we have a really aggressive future fork scheduled which changes the current behavior of the client.

Do you really need to know the max blob count of future forks or can you make it return the max blobs per block in the current or next epoch?

Reply:
This function is used as a "trick" to compensate the absence of the Python [defaultdict](https://docs.python.org/3/library/collections.html#collections.defaultdict).

It is used here:

```go
// returns a method with filters peers specifically for a particular blob subnet.
// All peers are supposed to be subscribed to all blob subnets.
func (s *Service) filterPeerForBlobSubnet() func(_ *enode.Node) (map[uint64]bool, error) {
	maxBlobsPerBlock := params.BeaconConfig().AbsoluteMaxBlobsPerBlock()
	result := make(map[uint64]bool, maxBlobsPerBlock)
	for i := range maxBlobsPerBlock {
		result[uint64(i)] = true
	}

	return func(_ *enode.Node) (map[uint64]bool, error) {
		return result, nil
	}
}
```

We need a way to say: "all peers are subscribed to all blob subnets".

Actually, blob subnets are quite a unique case. They are the only case where the number of subnets to join is not constant (moved at each fork between capella, deneb and electra) **and** where all peers should be subscribed to all of these subnets as well.

Reply:
Found a better solution here: 036459cbe633a5a41b8430334630063ff40d312d